plenum/server/consensus/consensus_shared_data.py

```diff
@@ -62,6 +62,14 @@ def __init__(self, name: str, validators: List[str], inst_id: int, is_master: bo
         self.primaries_batch_needed = False
         self.requestQueues = {}
 
+    @property
+    def last_ordered_3pc(self):
```

Comment:
Why does it need to be a property?

Reply:
it's for debug purposes and would be removed.